1-Draft/RFCXXXX-Policy.md

```diff
@@ -0,0 +1,234 @@
+---
+RFC:  RFCnnnn
+Author:  travisez13
+Status:  Draft
+SupercededBy:  N/A
+Version: 0.1
+Area: Engine
+Comments Due: 6/30/2019
+---
+
+# `PowerShell Core` Policy
+
+## Motivation
+
+Consumers, developers, and enterprise system administrators should be able to flexibly and reliable configure PowerShell 7.
+
+## Acknowledgement
+
+I based this off of @iSazonov 's RFC, for just a slightly different purpose.
+[PR #111](https://github.com/PowerShell/PowerShell-RFC/pull/111)
+
+## Goals
+
+1. Specify how PowerShell 7 will deal with having both Windows PowerShell and PowerShell Core Group Policy.
+    - This is covered in [Policy settings Setting Fall-Back](#policy-settings-setting-fall-back).
+1. Correct how the `pwsh -settingsfile` switch behaves.
+    - This is covered in  [Parameter `-settingsfile`](#Parameter--settingsfile)
+
+## Definitions
+
+- **Computer-Wide settings/policy**  - setting or policy applied to the entire operating system.
+- **User settings/policy** - setting or policy applied only to the user.
+
+## Specification
+
+`PowerShell 7` should be configured using the following schemes:
+
+- On Windows - Group Policy Objects (GPO), Group Policy Preferences (GPP) and settings files.
+- On Unix - settings files.
+
+The settings files have `JSON` format.
+
+**Warning** The settings files differ from `PowerShell 7` _profile_ files, which are PowerShell scripts run at startup.
+
+Configuration schemes allow to customize `PowerShell 7` in the most flexible way:
+
+- Enterprise system administrators can use GPO,
+    GPP and computer-wide settings files to apply approved configuration settings and mandatory security settings in a centralized manner.
+    Most settings can be applied either to the user or computer-wide.
+- Developers and consumers can use user, or computer-wide level setting files.
+
+### Configuration defaults
+
+PowerShell 7 has hard-coded defaults for all configuration options.
+
+The default values must be `secure-by-default`.
+
+For release versions hard-coded defaults must be the same as ones in pre-installed configuration files. For preview versions they may vary (ex., enable experimental features and so on).
+
+Computer-wide configuration includes security sensitive setting,
+and failing to read those setting could result in an insecure system.
+So, if during startup, PowerShell 7 cannot read files read from the Computer-Wide scope,
+it fails to startup.
+
+If during startup PowerShell 7 cannot read user configuration files it uses _hardcoded_ defaults.
+
+If during operation PowerShell 7 cannot read configuration files it continue to use _current_ (runtime) configuration values.
```

Comment:
I don't think I understand this. If policy files change during a specific session, PS7 will update to those new files and change behavior?

Reply:
Not a current behavior.  If it does, this is saying that updating the configuration is not a blocking operation.  I can cut this from the spec if you think it would clarify.  cc @iSazonov

Reply:
We also might not end up implementing every new specified behavior.  This for example could not be implemented without a significant issue.

Reply:
Yes, that's why I was confused. If you will not update behavior with all changes, then I don't think you should process them at all.

So instead of the current verbiage, perhaps "PowerShell 7 does not update configuration values from modified configuration files during a given session." That would certainly simplify life.

Reply:
```suggestion
PowerShell 7 does not update configuration values from modified configuration files during a given session after the configuration has been loaded.
```